src/modules/samples/cpp/src/so/SampleModule.cpp

```diff
@@ -50,6 +50,7 @@ int MmiGetInfo(
             {
                 OsConfigLogError(SampleLog::Get(), "MmiGetInfo(%s, -, %d) returned %d", clientName, *payloadSizeBytes, status);
             }
+            OSConfigTelemetryStatusTrace("payload", status);
```

Comment:
>OSConfigTelemetryStatusTrace [](http://example.com/codeflow?start=12&length=28)

We discussed to not instrument the sample module. Or do I remember wrong?

Reply:
i did not recall that. I can remove them though